### PR TITLE
input: kafka: Add initial documentation

### PR DESCRIPTION
**Pull request by tarruda**
Resolves https://github.com/fluent/fluent-bit/issues/7269

### CODE REVIEW

File: pipeline/inputs/kafka.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Kafka
+
+Kafka input plugin allows to subscribe to one or more topics and collect messages from an [Apache Kafka](https://kafka.apache.org/) service. This plugin use the official [librdkafka C library](https://github.com/edenhill/librdkafka) \(built-in dependency\)
```
Comment (patrick-stephens) [inline pipeline/inputs/kafka.md:3]: I like separate lines per sentence just to make it easier to review - you can only comment by line so if you put a whole paragraph on one line it's hard!

Comment (patrick-stephens) [inline pipeline/inputs/kafka.md:3]:
```suggestion
The Kafka input plugin allows subscribing to one or more Kafka topics to collect messages from an [Apache Kafka](https://kafka.apache.org/) service. This plugin uses the official [librdkafka C library](https://github.com/edenhill/librdkafka) \(built-in dependency\)
```

File: pipeline/inputs/kafka.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+## Configuration Parameters
+
+| Key | Description | default |
+| :--- | :--- | :--- |
+| brokers | Single of multiple list of Kafka Brokers, e.g: 192.168.1.3:9092, 192.168.1.4:9092. |  |
```
Comment (patrick-stephens) [inline pipeline/inputs/kafka.md:9]: typo `of` --> `or`

Comment (patrick-stephens) [reply pipeline/inputs/kafka.md:9]: Is it comma separated, what about spaces?

File: pipeline/inputs/kafka.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+| poll\_ms | Kafka brokers polling interval in milliseconds. | 500 |
+| rdkafka.{property} | `{property}` can be any [librdkafka properties](https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md) |  |
+
+## Getting Started
+
+In order to subscribe/collect messages from Apache Kafka, you can run the plugin from the command line or through the configuration file:
+
+### Command Line
+
+The **kafka** plugin, can read the parameters from the command line in two ways, through the **-p** argument \(property\), e.g:
```
Comment (patrick-stephens) [inline pipeline/inputs/kafka.md:22]: `in two ways`? There's only one way...

File: pipeline/inputs/kafka.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+$ fluent-bit -i kafka -o stdout -p brokers=192.168.1.3:9092 -p topics=some-topic
+```
+
+### Configuration File
+
+In your main configuration file append the following _Input_ & _Output_ sections:
+
+```text
+[INPUT]
+    Name        kafka
```
Comment (patrick-stephens) [inline pipeline/inputs/kafka.md:34]: How's tagging work?

Comment (tarruda) [reply pipeline/inputs/kafka.md:34]: I haven't added any special tag handling to in_kafka.

File: pipeline/inputs/kafka.md (hunk 3 above)
Comment (tarruda) [reply pipeline/inputs/kafka.md:22]: Copied from out_kafka, fixing in both...

File: pipeline/inputs/kafka.md (hunk 2 above)
Comment (tarruda) [reply pipeline/inputs/kafka.md]: AFAIK only comma-separated values are accepted.

File: pipeline/inputs/kafka.md (hunk 4 above)
Comment (patrick-stephens) [reply pipeline/inputs/kafka.md:35]: So presumably we should specify a tag then in the input for matching later or does it automatically pick up tags from something like forward?

**Review by patrick-stephens — APPROVED** [review]

File: pipeline/inputs/kafka.md (hunk 4 above)
Comment (patrick-stephens) [reply pipeline/inputs/kafka.md:35]: Forward should not - it should support overriding the tag provided but the forward protocol includes the tag.

**Pull request merged**